.github/actions/slack/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Slack'
+description: 'A simple action that send a slack message to a channel.'
+inputs:
+  message:
+    description: 'The message to send.'
```

Comment:
Maybe add some doc that a link to the run will be included in the message?

Reply:
That's a very good point, thanks!